sys/include/net/netopt.h

```diff
@@ -147,6 +147,8 @@ typedef enum {
                                  *   the current state */
     NETOPT_AUTOACK,             /**< en/disable link layer auto ACKs or read
                                  *   the current state */
+    NETOPT_ACK_PENDING,         /**< en/disable link layer auto ACKs with
```

Comment:
Could you refactor a little bit this comments? Just to make the feature more clear